app/controllers/catalog_controller.rb

```diff
@@ -180,11 +180,11 @@ def default_solr_doc_params(id = nil)
   end
 
   def show
-    params[:id] = 'druid:' + params[:id] unless params[:id].include? 'druid'
-    @obj = Dor.find params[:id]
-
+    @obj = find_druid(params[:id])
+    return if @obj.nil?
```

Comment:
now that `find_druid` would throw an `ObjectNotFoundError` if the requested object isn't found, i think these `return if @obj.nil` lines are no longer necessary.